src/ml_gym/frontend/dashboard/src/features/modelEvaluations/modelEvaluationsSlice.tsx

```diff
@@ -0,0 +1,37 @@
+import { createSlice } from '@reduxjs/toolkit'
+import { ModelEvaluationMessageCollectionType, ModelEvaluationType } from '../../app/datatypes'
+import type { RootState } from '../../app/store';
+
+const initialState = {
+    messages: [],
+} as ModelEvaluationMessageCollectionType
+
+
+
+
+const modelEvaluationsSlice = createSlice({
+    name: 'modelEvaluations',
+    initialState,
+    reducers: {
+        modelEvaluationAdded(state, action) {
```

Comment:
Codacy has a fix for the issue: [Delete `····`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=9966992)
```suggestion
    modelEvaluationAdded(state, action) {
```